crates/iroha_crypto/src/merkle.rs

```diff
@@ -119,7 +119,7 @@ trait CompleteBinaryTree {
 
     /// Returns the index of the sibling node of the node at `index`, if it exists.
     fn sibling_index(&self, index: usize) -> Option<usize> {
-        if index.is_multiple_of(2) {
+        if index % 2 == 0 {
```

Comment:
I’m not too particular about it, but is it still not stabilized yet?
https://doc.rust-lang.org/std/primitive.usize.html#method.is_multiple_of

Reply:
Oh, indeed, it was stabilized in `1.87`. I am at `1.86` in Explorer.

I can bump and we can close this PR. Or we can merge it anyway, as it slightly relaxes unnecessary version requirement.

Reply:
Let's just quickly merge this.